test/models/report_test.rb

```diff
@@ -20,12 +18,12 @@ def setup
   end
 
   test '時間を表示する' do
-    assert_equal 'Wed, 03 Jan 2024'.to_date, @report.created_on
+    assert_equal Time.current.to_date, @report.created_on
   end
 
   test 'メンションする日報を取得する' do
-    include_url_report = Report.new(user: @takeru, title: 'URLを含む日報', content: "http://localhost:3000/reports/#{@report.id}, よっちゃんの記事")
-    include_url_report.save
-    assert_equal @report, include_url_report.mentioning_reports[FIRST_REPORT]
+    other_report = create(:report, user: @takeru, content: "http://localhost:3000/reports/#{@report.id}, よっちゃんの記事")
+    other_report.save!
+    assert_equal @report, other_report.mentioning_reports.first
```

Comment:
```suggestion
    assert_equal [@report], other_report.mentioning_reports
```

firstでチェックすると「もしかして2つ目や3つ目のreportがあるのでは？」と思われそうなので、配列として比較する方がベターかも